app/controllers/api/saved_jobs_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::SavedJobsController < ApplicationController
+  before_action :authorize!, only: [:index, :show]
+
+  def index
+    jobs_to_look_up = current_user.saved_jobs
+    jobs_to_send = look_up(jobs_to_look_up)
+    render json: jobs_to_send
+  end
+
+  def show
+    job = IndeedApi.find_job(params[:job_key])
+    if job
+      job_to_update = Job.find_by(job_key: params[:job_key])
+      job_output = info_for_output(job)
+      job_to_update.update_job(job_output)
+      render json: job_output
+    else
+      render json: {message: "Sorry, this job is no longer available", status: :not_found}
+    end
+  end
+
+  private
+
+  def info_for_output(job_from_indeed)
```

Comment:
This code looks to be duplicated do we need it in both the model and the controller?